### ISSUE TITLE
Testing the odometer algorithm

### ISSUE THREAD

**Issue by RichRick1** [theochem/PyCI#36]
@alnaba1, to test the odometer algorithm we can use H2 and H4 systems in the STO-3G basis set. By providing different cost function we can see which determinants are selected.
We can use pyscf package to generate .fcidump file and test the results on it:

```
from pyscf import gto
mol = gto.Mole()
mol.build( atom = "H 0 0 0; H  0 1 0", basis = 'sto-3g')
```

```
# Import modules
import pyci

# System information
filename = datafile("lih_sto6g.fcidump")
ham = pyci.hamiltonian(filename)
```

@PaulWAyers do you think it's good testing systems?

**Comment by PaulWAyers** [comment]
Is there any update on this @RichRick1 or @alnaba1 ?

**Comment by alnaba1** [comment]
@PaulWAyers It's been finished for a while, there is a PR that I'm waiting on to be merged #55

**Comment by alnaba1** [comment]
Merged #55, closing issue.
